lineapy/api/api.py

```diff
@@ -278,6 +280,18 @@ def get(artifact_name: str, version: Optional[int] = None) -> LineaArtifact:
         date_created=artifact.date_created,  # type: ignore
     )
 
+    # Check version compatibility
+    system_python_version = float(
+        f"{sys.version_info.major}.{sys.version_info.minor}"
+    )
```

Comment:
extract as a util function. there is some level of matching what we do here with what was saved when session context was created. if we change the float to a string for eg, the two need to match. best if we extract this piece into a small `get_system_python_version` function or something that returns a float/str whatever we decide.

Reply:
These are great points, thanks! Working to update accordingly.